local-dev.html

```diff
@@ -12,16 +13,13 @@
 </head>
 <body>
     <p>
-        <button type="button" id="loadAFconfig">Load af database + server</button>
-        <button type="button" id="elementPath">Load af database + server</button>
-        <button type="button" id="getEFs">get EFs off element</button>
-        <button type="button" id="getTemplateAttributes">get attributes</button>
-        <button type="button" id="buildTreemap">Build treemap</button>
+        <button type="button" id="btnUpdate">Update</button>
```

Comment:
Thanks for removing all these buttons! (They were good at the time, but it's nice to see them gone)